### ISSUE TITLE
Option to do it in batch

### ISSUE THREAD

**Issue by HitogiDev** [kit-nya/anki_furigana#4]
Is there an option to add the definition in batch?

**Comment by kit-nya** [comment]
Yeah, I suppose I can add this in! I was looking to add in a Japanese dictionary anywho, so this would work well with this. Thanks for the feedback.

**Comment by kit-nya** [comment]
Hi again!

This has been added and updated on anki addons. Thanks for the suggestion!
